.gitignore

```diff
@@ -133,3 +133,10 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+<<<<<<< Updated upstream
+.DS_Store
+.DS_Store
+viz_scripts/.DS_Store
+=======
+viz_scripts/auxiliary_files/.DS_Store
+>>>>>>> Stashed changes
```

Comment:
This is a merge conflict. You need to resolve it properly before committing

Reply:
@styoung3 the reason that you see `.DS_Store` over and over again is that your `.gitignore` is broken due to merge conflicts.